packages/lib/ren-tx/src/configs/genericMint.ts

```diff
@@ -195,6 +196,7 @@ const depositListener = (
                                               }),
                                     )
                                     .catch((e) => {
+                                        console.error("Sign error!", e);
```

Comment:
It might make sense to use a logging library to give devs control over the various error logs in ren-tx.

Reply:
Will add this at a later date as an improvement, as to not block Tom

Reply:
Agreed - that wasn't really feedback about this PR in particular, just of ren-tx.